doc/reference/guidelines.rst

```diff
@@ -0,0 +1,48 @@
+.. highlight:: python
+
+=================
+Coding Guidelines
+=================
+
+Python
+======
+
+- PEP8 ignore E501
+- prefer % over format
```

Comment:
The other way around is significantly more flexible, and more practical for translations (AFAIK Python's sprintf does not support explicit indexes so only named format parameters can be reordered in translations)